docs/conf/australia/2023/news/welcome.rst

```diff
@@ -0,0 +1,83 @@
+
+The conference will take place during day time in AEDT_.
+
+.. _AEDT: https://time.is/AEDT
+
+Our venue: Library at the Dock
+--------------------------------
+
+Our Australian conference will be held at `Library at the Dock`_. 
+It is a beautiful library space located alongside the Yarra river, at the Docklands precinct of Melbourne.
+
+.. image:: /_static/conf/images/pics/Lib-Dock.jpg
+
+Library at The Dock is Australia’s first 6 star Green Star rated public building and we will have access to these spaces:
+
+* The Performance space where we will be hosting our conference talks.
+* The Community Room which will host the unconference sessions, and where our snacks and drinks catering will happen.
+
+It also has *outdoor seating* and a *park* directly on the property for plenty of fresh air and free space.
+
+.. _Library at the Dock: https://www.melbourne.vic.gov.au/community/hubs-bookable-spaces/the-dock/library-at-the-dock/Pages/library-at-the-dock.aspx
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!
+
+We plan to launch our CFP early to mid June and have it last until sometime until mid-September.
+Get started coming up with ideas for the conference now.
+
+Tickets
+-------
+
+Tickets will officially go on sale in **July {{ year }}**.
+We will announce ticket prices soon, so that you can start planning your budget.
+As with the previous years, we have tiered pricing to make it fair for everyone.
+
+As always, we keep ticket prices low so that it is accessible to the widest range of people possible.
+However, if you can't afford these prices and still wish to attend,
+we will bringing our Opportunity Grants program back again this year.
+
+Sponsorship
+-----------
+
+We are planning to do similar sponsorship levels again this year as last year.
+In addition to the regular sponsorship levels, we have additional ways to sponsor our online events.
+See our `sponsorship prospectus <https://www.writethedocs.org/conf/australia/2023/sponsors/prospectus/>`_ for all the options.
+
+We're always open to customisations on our sponsorships, especially given the new online format.
+You can always reach us with ideas or questions at sponsorship@writethedocs.org.
+
+Stay Updated
+------------
+
+Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?
+Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third-parties, and you can unsubscribe at any time.
```

Comment:
Why is this a URL shortner? @mxsasha any idea? Looks like it has been for years.